<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SideWidget</class>
 <widget class="QWidget" name="SideWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>366</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>269</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="Transformations">
    <attribute name="title">
     <string>Transformations</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>271</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Transformations</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>21</y>
        <width>261</width>
        <height>230</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Rotate around x</string>
         </property>
         <property name="buddy">
          <cstring>rotate_x_slider</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="rotate_x_slider">
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Rotate around y</string>
         </property>
         <property name="buddy">
          <cstring>rotate_y_slider</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="rotate_y_slider">
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Rotate around z</string>
         </property>
         <property name="buddy">
          <cstring>rotate_z_slider</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="rotate_z_slider">
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Zoom factor</string>
         </property>
         <property name="buddy">
          <cstring>zoom_factor_spin_box</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="zoom_factor_spin_box">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>0.000100000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Translate along x</string>
         </property>
         <property name="buddy">
          <cstring>trans_x_spin_box</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="trans_x_spin_box">
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Translate along y</string>
         </property>
         <property name="buddy">
          <cstring>trans_y_spin_box</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="trans_y_spin_box">
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Translate along z</string>
         </property>
         <property name="buddy">
          <cstring>trans_z_spin_box</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="trans_z_spin_box">
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Rendering_options">
    <attribute name="title">
     <string>Rendering options</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_render">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Render</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_render_joints">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Render joints</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_render_links">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Render links</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_render_mesh">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Render mesh</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Manage_skeleton">
    <attribute name="title">
     <string>Manage Skeleton</string>
    </attribute>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Joint</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="buddy">
      <cstring>trans_x_spin_box</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>106</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
     <property name="buddy">
      <cstring>trans_y_spin_box</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>132</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>z</string>
     </property>
     <property name="buddy">
      <cstring>trans_z_spin_box</cstring>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
